workshops/rock_paper_scissors/README.md

```diff
@@ -0,0 +1,274 @@
+---
+Name: 'Rock Paper Scissors Game'
+Description: 'Easy rock paper scisoors game using Python'
+Author: '@JackTDC'
+---
+
+# Rock Paper Scissor Game
+
+Have you ever thought about building a game with Python? Well today is your lucky day.
+
+So let's get started.
+
+![homepage](https://cloud-8p13u30yt.vercel.app/rps.png)
+
+Here's the [live demo][final_live_demo] and [final code][final_code].
+
+[final_live_demo]:https://repl.it/@JackTDC/Rock-paper-Scissors#main.py
+[final_code]: https://repl.it/@JackTDC/Rock-paper-Scissors#main.py
+[repl_it]: https://repl.it
+
+## Part 1: Prerequisites
+
+You should have a beginner understanding of: 
+- Python
+
+## Part 2: Setup
+
+### Setting up your code environment on Repl.it
+
+[Repl.it](https://repl.it) is an online code editor where you can build your game. You don't have to use Repl.it but I suggest you do as it sets everything up for you and you don't require any installations.
+
+To get started, go to [https://repl.it/languages/python](https://repl.it/languages/python). Your coding environment will spin up in just a few seconds!
+
+You should see something like the following:
+
+![Image](https://cloud-lukrtynqs.vercel.app/repl.png)
+
+## Part 3: Making The Game
+![Image](https://cloud-931ehqeec.vercel.app/coding.gif)
+
+### 1)Importing Modules
+Let us start making our game! First, we'll need to import some modules
+So in the first line type `from random import randint`. This will import the radient module
+
+### 2)Giving values to the variables
+First let's make a list for all th entries a plyer could enter,
+`t = ["r","p","s"]`
+
+Now we have to write a command so the computer chooses rock, paper or scissors randomly!
+`computer = t[randint(0,2)]`.
```

Comment:
```suggestion
```py
computer = t[randint(0,2)]
```
```

Reply:
I didn't understand the change? can you explain

Reply:
As it is a ONE LINE code, so i have suggested that u make it a code block. Also, the 'py' written there is syntax highlighting.